tests/test_critical_path_analysis.py

```diff
@@ -0,0 +1,265 @@
+import gzip
+import json
+import os
+import unittest
+from collections import Counter
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Tuple
+
+from hta.analyzers.critical_path_analysis import CPEdge, CPEdgeType
+from hta.trace_analysis import TraceAnalysis
+
+
+class CriticalPathAnalysisTestCase(unittest.TestCase):
```

Comment:
thanks for seperating this out, assuming these are identical to previous test in test_trace_analysis so shoudl be good :)

Reply:
They are the same. With separate test files, it easy to debug and test.